subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -361,8 +361,8 @@
 
```

Comment:
I'm missing a general pointer towards the `java-library` plugin, which is what users will want to use when their project is a library (which is the 95% case).